### PR TITLE
PAYARA-2378 JSON Parsing issues

### PR DESCRIPTION
**Pull request by Cousjava**

### CODE REVIEW

**Comment by Cousjava** [comment]
Jenkins, quick test Payara 5 please

**Comment by Cousjava** [comment]
Jenkins, quick test Payara 5 please

File: nucleus/admin/util/src/main/java/com/sun/enterprise/admin/remote/reader/ProgressStatusDTOJsonProprietaryReader.java [hunk 1]
```diff
@@ -67,8 +69,7 @@ public ProgressStatusDTO readFrom(final HttpURLConnection urlConnection) throws
 
     @Override
     public ProgressStatusDTO readFrom(final InputStream is, final String contentType) throws IOException {
-        JsonParser jp = factory.createJsonParser(is);
-        try {
+        try (JsonParser jp = factory.createJsonParser(is)) {
```
Comment (arjantijms) [inline nucleus/admin/util/src/main/java/com/sun/enterprise/admin/remote/reader/ProgressStatusDTOJsonProprietaryReader.java]: Just as a comment, but when you see uncommon abbreviations like "jp" here, best to rename it to something more clear right away

**Comment by Cousjava** [comment]
Jenkins, quick test Payara 5 please

**Comment by arjantijms** [comment]
Jenkins, quick test Payara 5 please

**Comment by Cousjava** [comment]
Jenkins, quick test Payara 5 please

**Comment by Cousjava** [comment]
Jenkins, quick test Payara 5 please

**Comment by Cousjava** [comment]
Jenkins, quick test Payara 5 please

**Comment by arjantijms** [comment]
Jenkins, quick test Payara 5 please

**Pull request merged**